pprof_test.go

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 CloudWeGo Authors
```

Comment:
No need update all - only modified file

Reply:
get

Reply:
When I changed it back to 2022, the corresponding file failed the CI test (which is why I changed the existing file to 2023 in the first place, I checked `.licenserc.yaml` and it seemed fine).
![img_v2_00a6bdb3-99e5-4d2d-8ddf-e367375af6eg](https://user-images.githubusercontent.com/28105615/232663774-3469a9f2-895d-4520-af04-1420c0395b98.jpg)

Reply:
https://github.com/cloudwego/hertz/blob/develop/.github/workflows/pr-check.yml#L12 
Modify this to align the hertz main library and it should be fine.

Reply:
Got it.thanks